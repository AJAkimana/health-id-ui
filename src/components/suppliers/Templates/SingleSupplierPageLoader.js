import React from 'react';
import ContentLoader from 'react-content-loader';

export const SingleSupplierPageLoader = () => (
  <ContentLoader
    height={1600}
    width={600}
    speed={1}
    primaryColor="#cccccc"
    secondaryColor="#ecebeb"
  >
    <rect x="100" y="50" rx="2" ry="2" width="400" height="20" />
    <rect x="140" y="80" rx="2" ry="2" width="320" height="13" />
    <rect x="140" y="110" rx="3" ry="3" width="80" height="5" />
    <rect x="140" y="120" rx="3" ry="3" width="160" height="5" />
    <rect x="140" y="130" rx="3" ry="3" width="80" height="5" />
    <rect x="140" y="140" rx="3" ry="3" width="160" height="5" />
    <rect x="140" y="160" rx="3" ry="3" width="240" height="5" />
    <rect x="390" y="100" rx="100" ry="100" width="70" height="70" />
    <rect x="140" y="187" rx="0" ry="0" width="320" height="13" />
    <rect x="140" y="262" rx="0" ry="0" width="320" height="13" />
    <rect x="149" y="215" rx="0" ry="0" width="51" height="5" />
    <rect x="230" y="215" rx="0" ry="0" width="59" height="5" />
    <rect x="324" y="215" rx="0" ry="0" width="57" height="5" />
    <rect x="150" y="233" rx="0" ry="0" width="60" height="1" />
    <rect x="230" y="233" rx="0" ry="0" width="60" height="1" />
    <rect x="325" y="233" rx="0" ry="0" width="60" height="1" />
    <rect x="140" y="290" rx="0" ry="0" width="320" height="97" />
  </ContentLoader>
);

export default SingleSupplierPageLoader;
