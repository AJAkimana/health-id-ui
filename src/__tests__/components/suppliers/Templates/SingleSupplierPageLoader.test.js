import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SingleSupplierPageLoader from '../../../../components/suppliers/Templates/SingleSupplierPageLoader';

const props = {
  variables: {},
  classes: {},
  session: {
    me: {
      role: {
        name: 'Master Admin'
      },
      activeOutlet: {
        outletpreference: {
          outletTimezone: {
            name: "Africa/Nairobi"
          }
        }
      }
    }
  }
};

describe('SingleSupplierPageLoader', () => {
  it('should render the SingleSupplierPageLoader component properly', () => {
    const setupSingleSupplierPageLoader = () => {
      const wrapper = shallow(<SingleSupplierPageLoader {...props} />)
      return wrapper
    }

    let wrapper;
    wrapper = setupSingleSupplierPageLoader();
    expect(toJson(wrapper)).toMatchSnapshot();
  })
});
