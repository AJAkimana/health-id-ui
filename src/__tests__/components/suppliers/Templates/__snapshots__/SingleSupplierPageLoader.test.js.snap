// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SingleSupplierPageLoader should render the SingleSupplierPageLoader component properly 1`] = `
<ContentLoader
  height={1600}
  primaryColor="#cccccc"
  secondaryColor="#ecebeb"
  speed={1}
  width={600}
>
  <rect
    height="20"
    rx="2"
    ry="2"
    width="400"
    x="100"
    y="50"
  />
  <rect
    height="13"
    rx="2"
    ry="2"
    width="320"
    x="140"
    y="80"
  />
  <rect
    height="5"
    rx="3"
    ry="3"
    width="80"
    x="140"
    y="110"
  />
  <rect
    height="5"
    rx="3"
    ry="3"
    width="160"
    x="140"
    y="120"
  />
  <rect
    height="5"
    rx="3"
    ry="3"
    width="80"
    x="140"
    y="130"
  />
  <rect
    height="5"
    rx="3"
    ry="3"
    width="160"
    x="140"
    y="140"
  />
  <rect
    height="5"
    rx="3"
    ry="3"
    width="240"
    x="140"
    y="160"
  />
  <rect
    height="70"
    rx="100"
    ry="100"
    width="70"
    x="390"
    y="100"
  />
  <rect
    height="13"
    rx="0"
    ry="0"
    width="320"
    x="140"
    y="187"
  />
  <rect
    height="13"
    rx="0"
    ry="0"
    width="320"
    x="140"
    y="262"
  />
  <rect
    height="5"
    rx="0"
    ry="0"
    width="51"
    x="149"
    y="215"
  />
  <rect
    height="5"
    rx="0"
    ry="0"
    width="59"
    x="230"
    y="215"
  />
  <rect
    height="5"
    rx="0"
    ry="0"
    width="57"
    x="324"
    y="215"
  />
  <rect
    height="1"
    rx="0"
    ry="0"
    width="60"
    x="150"
    y="233"
  />
  <rect
    height="1"
    rx="0"
    ry="0"
    width="60"
    x="230"
    y="233"
  />
  <rect
    height="1"
    rx="0"
    ry="0"
    width="60"
    x="325"
    y="233"
  />
  <rect
    height="97"
    rx="0"
    ry="0"
    width="320"
    x="140"
    y="290"
  />
</ContentLoader>
`;
